cmd/sync/sync.go

```diff
@@ -24,7 +24,7 @@ var (
 	dryRun    = flag.Bool("dry-run", false, "Print resources to be synced instead of mutating cluster state.")
 	once      = flag.Bool("run-once", false, "If true, run only once then quit.")
 	interval  = flag.Duration("interval", 3*time.Minute, "How often the sync process going to be rerun.")
-	logLevel  = flag.String("loglevel", "Debug", "valid values are Debug, Info, Warning, Error")
+	logLevel  = flag.String("loglevel", "Info", "valid values are Debug, Info, Warning, Error")
```

Comment:
would like to see this made the default for all our containers - @mjudeikis perhaps you can pick that up in #1464?